metricbeat/module/apache/status/status_test.go

```diff
@@ -178,12 +178,12 @@ func TestFetchTimeout(t *testing.T) {
 	elapsed := time.Since(start)
 	var found bool
 	for _, err := range errs {
-		if strings.Contains(err.Error(), "request canceled (Client.Timeout exceeded") {
+		if strings.Contains(err.Error(), "context deadline exceeded (Client.Timeout exceeded") {
```

Comment:
maybe for another PR, but better check for `err == context.X`. There are constants for this. Checking errors by error string is kind of a no-go.